stable/contour/README.md

```diff
@@ -0,0 +1,101 @@
+# Contour
+
+[Contour](https://github.com/heptio/contour) is an Ingress controller for Kubernetes that works by deploying the Envoy proxy as a reverse proxy and load balancer. Unlike other Ingress controllers, Contour supports dynamic configuration updates out of the box while maintaining a lightweight profile.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release .
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release --purge
+```
+## Upgrading the Chart
+
+```bash
+$ helm upgrade --install my-release .
+```
+
+
+## Configuration
+
+The default configuration values for this chart are listed in `values.yaml`.
+
+| Parameter                             | Description                                                  | Default                                           |
+|---------------------------------------|-------------------------------------                         |---------------------------------------------------|
+| `deployment.replicas`                 | Number of replica sets                                       | 2                                                 |
+| `contour.image.repository`            | Repository for contour container image                       | gcr.io/heptio-images/contour                      |
+| `contour.image.tag`                   | Contour image tag                                            | master                                            |
```

Comment:
The version of the image is in the values file as a tag. Can you please update this instead of noting `master`.

Reply:
Added contour version instead of master